iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_queue_popper.hpp

```diff
@@ -94,8 +84,10 @@ class ChunkQueue
     MemberType_t* getMembers() noexcept;
 
   private:
-    MemberType_t* m_chunkQueueDataPtr{nullptr};
+    MemberType_t* const m_chunkQueueDataPtr;

```

Comment:
again, the `* const` implicitly deletes the defaulted move assignment operator

Reply:
OK, so the best we can do to be moveable and the better API would then be a reference in the c'tor and the non const pointer as member. Most of these objects will be created with a ...Data object that was created before and can then be passed as reference